src/App.jsx

```diff
@@ -0,0 +1,42 @@
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  //  Link,
+} from 'react-router-dom';
+import './App.scss';
+import Header from './components/Header/Header';
+import BooksPage from './pages/Books';
+import CategoriesPage from './pages/Categories';
+
+const links = [
+  {
+    path: '/',
+    text: 'Books',
+    element: (<BooksPage />),
+  },
+  {
+    path: '/categories',
+    text: 'Categories',
+    element: (<CategoriesPage />),
+  },
+];
```

Comment:
Has 2 `<Link>`s in the navbar which leads to the booklist and categories locations 🤓